README.md

```diff
@@ -100,6 +100,91 @@ cabal test all
 Alternatively you can setup [`lorri`](https://github.com/target/lorri) so that
 cabal is available without needing to enter the nix shell.
 
+## Integration with Cardano
+
+TODO: consider linking a video of the demo. Maybe at the beginning of this README.
+
+The update protocol implemented in this repository was integrated into Cardano,
+although this was not merged into the `master` branch of any of the its
+components, due to the experimental nature of this work. This integration took
+place across several components:
+
+- Ledger, hosted in [`cardano-ledger-specs`](). The integration can be found in commit ...
+- Consensus
+- Node and Cardano command line interface,
+- Devops infrastructure
+
+In particular, `cardano-ops` contains a script for testing the integration, and
```

Comment:
Need to provide a link to the repo and where is the script located and with what name.